tests/e2e/tests/ChatSection/TriggerStateShouldHaveWarningEvents.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { expect, test } from '@playwright/test';
+import AggregatesComponent from 'Pages/AggregatesComponent';
+import ChatComponent from 'Pages/ChatComponent';
+import DashboardPage from 'Pages/DashboardPage';
+import UserStateComponent from 'Pages/UserStateComponent';
+import { qase } from 'playwright-qase-reporter';
+import { TriggerDataSet } from 'testData/testData.enum';
+
+import {
+  getAccessToken,
+  mockFloods,
+  resetDB,
+} from '../../helpers/utility.helper';
+import LoginPage from '../../Pages/LoginPage';
+
+let accessToken: string;
+
+test.beforeEach(async ({ page }) => {
+  // Login
+  const loginPage = new LoginPage(page);
+
+  accessToken = await getAccessToken();
+  await resetDB(accessToken);
+  // We should maybe create one mock for all different disaster types for now we can just use floods
+  await mockFloods(
+    TriggerDataSet.TriggerScenario,
+    TriggerDataSet.CountryCode,
+    accessToken,
+  );
+
+  await page.goto('/');
+  await loginPage.login(TriggerDataSet.UserMail, TriggerDataSet.UserPassword);
+});
+
+test(
+  qase(
+    43,
+    '[Trigger] Event list should show at least 1 trigger, medium or low warning event',
+  ),
+  async ({ page }) => {
+    const aggregates = new AggregatesComponent(page);
+    const dashboard = new DashboardPage(page);
+    const userState = new UserStateComponent(page);
+    const chat = new ChatComponent(page);
+
+    // Navigate to disaster type the data was mocked for
+    await dashboard.navigateToFloodDisasterType();
+    // Assertions
+    await userState.headerComponentIsVisible({
+      countryName: TriggerDataSet.CountryName,
+    });
+    await chat.chatColumnIsVisibleForTriggerState({
+      firstName: TriggerDataSet.firstName,
+      lastName: TriggerDataSet.lastName,
+    });
+    await chat.allChatButtonsArePresent();
+
+    // get the number of warning events and aggregated events
+    const warningCount = await chat.chatPredictionButtonsAreActive();
+    const aggregatesNumber = await aggregates.getNumberOfPredictedEvents();
```

Comment:
I think it's good to connect nr of events in chat with nr of events in aggregates section, but that is something else then what the test description above says. That doesn't talk about aggregates section.

Reply:
Updated description.